libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/Matrix/MatrixFooter/MatrixFooter.tsx

```diff
@@ -32,13 +34,13 @@ export class MatrixFooter extends React.PureComponent<IMatrixFooterProps> {
                 key={`${this.props.matrixLength}_${0}category1`}
                 className={classNames.matrixCellPivot1Categories}
               />
-              {this.props.category1Values.map((category: any, i: number) => {
+              {this.props.category1Values.map((_: any, i: number) => {
```

Comment:
this part seems making more sense to extract as a react component.
so it can be used everywhere formatCategory is referenced.

formatCategory is more like a displaying processor, which should be better to sits in an tsx file

Reply:
done, made formatCategory a class/tsx file